src/vm/contexts.rs

```diff
@@ -186,6 +240,26 @@ impl AssetMap {
             map.insert(principal, output_map);
         }
 
+        for (principal, stx_amount) in self.stx_map.drain() {
+            let output_map = if map.contains_key(&principal) {
+                map.get_mut(&principal).unwrap()
+            } else {
+                map.insert(principal.clone(), HashMap::new());
+                map.get_mut(&principal).unwrap()
+            };
+            output_map.insert(AssetIdentifier { contract_identifier: QualifiedContractIdentifier::transient(), asset_name: ClarityName::from("STX") }, AssetMapEntry::STX(stx_amount as u128));
```

Comment:
We should probably have a built-in contract identifier for STX, rather than using the transient identifier.

Reply:
How about all 0's?  That's the contract address of the chain's "boot code" (see `src/chainstate/stacks/db/mod.rs`).

Reply:
Sounds good to me